### ISSUE TITLE
dependency-updates report not found when using `reportfileName`

### ISSUE THREAD

**Issue by memelet** [patrikerdes/gradle-use-latest-versions-plugin#33]
I configure dependency-updates as
```
dependencyUpdates {
  outputDir = "$buildDir/reports"
  reportfileName = "dependency-updates"
  ...
}
```
which results in 
```
java.nio.file.NoSuchFileException: /Users/bkaplan/si/es-rmb-core/build/dependencyUpdates/report.json
```

**Comment by memelet** [comment]
Yea, I looks like the report file is hard-coded in several places
```
new File(project.buildDir, 'dependencyUpdates' + File.separator + 'report.json')
```

**Comment by patrikerdes** [comment]
Yep. looks like a bug

**Comment by usr42** [comment]
I'm going to pick that up.

**Comment by usr42** [comment]
The PR https://github.com/patrikerdes/gradle-use-latest-versions-plugin/pull/36 should fix this issue.

**Comment by usr42** [comment]
This should be fixed now.

**Comment by patrikerdes** [comment]
Released in 0.2.13
